components/Footer/styles.css

```diff
@@ -1,192 +1,228 @@
 /** @define Footer; use strict */
 
+.button-primary {
+  border: none;
+  outline: none;
+  display: inline-block;
+  width: 100%;
+  font-size: 14px;
+  font-weight: 900;
+  color: #FFFFFF;
+  letter-spacing: 1.5px;
+  max-width: 300px;
+  height: 44px;
+  line-height: 43px;
+  text-align: center;
+  position: relative;
+  text-transform: uppercase;
+  background: #F8AA31;
+  -webkit-border-radius: 3px;
+  -moz-border-radius: 3px;
+  border-radius: 3px;
+  transition: all 200ms ease-in;
+  cursor: pointer; }
+
+.button-primary:hover {
+  background-color: #f7a018;
+  color: #FFFFFF;
+}
+
 .Footer {
+  margin: 8px;
   background: #393939;
-  box-sizing: border-box;
-  bottom: 0;
-  color: #fff;
-  font-size: 0.666em;
-  padding: 3em 5% 1.5em;
-  text-align: left;
-  text-transform: uppercase;
+  padding: 48px;
+}
+
+.Footer-description {
+  font-size: 16px;
+  line-height: 1.5em;
+  font-weight: 300;
+  padding-bottom: 32px;
+}
+
+.Footer-description a {
+  color: #F8AA31;
+}
+
+.Footer-input-container {
+  position: relative;
+  margin-bottom: 24px;
+}
+
+.Footer-input-container input {
```

Comment:
add a `box-sizing: border-box;` to this class, the input is currently peaking out of the 300px restriction